v3/src/components/tool-shelf/help-button.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import { Menu, MenuButton, MenuItem, MenuList } from "@chakra-ui/react"
+import HelpIcon from "../../assets/icons/icon-help.svg"
+import { useDocumentContent } from "../../hooks/use-document-content"
+import { getDefaultLanguage, t } from "../../utilities/translation/translate"
+import { kWebViewTileType } from "../web-view/web-view-defs"
+import { isWebViewModel } from "../web-view/web-view-model"
+import { kRightButtonBackground, ToolShelfButtonTag } from "./tool-shelf-button"
+import { logMessageWithReplacement } from "../../lib/log-message"
+
+import "./tool-shelf.scss"
+
+const showHelpURL = 'https://codap.concord.org/help'
+const showHelpURL_ja = 'https://codap.concord.org/resources/latest/help-documents/CODAP解説書.pdf'
+const showHelpForumURL = 'https://codap.concord.org/forums/forum/test/'
+const showWebSiteURL = 'https://codap.concord.org'
+const showPrivacyURL = 'https://codap.concord.org/privacy'
+
+const helpURLs: Record<string, string> = {
+  "ja": showHelpURL_ja
+}
+
+export const HelpShelfButton = () => {
+  const documentContent = useDocumentContent()
+
+  const handleShowHelp = () => {
+    const locale = getDefaultLanguage()
+    const url = helpURLs[locale]
+
+    if (url !== undefined) {
+      documentContent?.applyModelChange(() => {
+        const tile = documentContent?.createOrShowTile?.(kWebViewTileType)
+        isWebViewModel(tile?.content) && tile?.content.setUrl(`${url}`)
+      }, {
+        undoStringKey: "V3.Undo.webView.show",
+        redoStringKey: "V3.Redo.webView.show",
+        log: logMessageWithReplacement("Show web view: %@", {url}, "document")
+      })
```

Comment:
This code is duplicated from lines 27-34 of `options-button.tsx`. Instead of duplicating code, you should make a utility function to do this, and then call that utility function here and in `options-button.tsx`.